mempool/mempool
# Mining dashboard still runs fine if Bitcoin Core becomes unavailable

Fixes https://github.com/mempool/mempool/issues/1705

Depends on https://github.com/mempool/mempool/pull/1696 which must be merged first to avoid conflicts, as this branch is forked from `nymkappa/feature/merge-blocks`

#### Testing

* Run bitcoind
* Run mempool backend + frontend
* Wait a couple of seconds for things to start
* Stop bitcoind
* Open the browser network tab and disable the cache
* Browse mining dashboard pages with ease(tm). You will notice some value being zeroed.
